js/main.js

```diff
@@ -0,0 +1,127 @@
+$(document).ready(function () {
+
+    $("#loginForm").submit(function (event) {
+
+        var email = $("#email").val();
+        var password = $("#pwd").val();
+
+        $.ajax({
+            type: "POST",
+            data: {
+                action: 'authenticateUser',
+                table: 'test_table1',
+                email: email,
+                password: password
+            },
+            url: "ajax_php/actions.php",
+            beforeSend: function () {
+                console.log('sending request ...');
+            },
+            dataType: "html",
+            async: true,
+            success: function (data) {
+
+                var result = JSON.parse(data);
```

Comment:
1) daca-i pui "dataType": "json", jQuery iti face automat JSON.parse la date.
2) orice raspuns de la server pe care-l parsezi ca si JSON manual (JSON.parse...) il pui intr-un try { } catch {}, si in catch tratezi eventualele erori